src/DatePicker/DatePickerDialog.js

```diff
@@ -115,6 +115,7 @@ class DatePickerDialog extends Component {
     const styles = {
       dialogContent: {
         width: mode === 'landscape' ? 479 : 310,
+        userSelect: 'none',
```

Comment:
Can you apply this property as close as where it's needed?
That seems perfect here https://github.com/callemall/material-ui/blob/master/src/DatePicker/Calendar.js#L247.